content/tutorials/native-aa-multisig/10.index.md

```diff
@@ -891,13 +938,26 @@ Make sure you deposit funds on ZKsync Era using [one of the available bridges](h
     }
     ```
 
+    :test-action{actionId="deploy-script-pk"}
+
 1. From the project root, compile and deploy the contracts.
 
-    ```sh
+    :test-action{actionId="compile-and-deploy-factory"}
+
+    ::code-group
+
+    ```shell [npx]
+    npx hardhat compile
+    npx hardhat deploy-zksync --script deploy-factory.ts
```

Comment:
Need to update the import statement from `import { Deployer } from "@matterlabs/hardhat-zksync-deploy";` to `import { Deployer } from "@matterlabs/hardhat-zksync";`